jcl/src/java.base/share/classes/java/lang/invoke/MethodType.java

```diff
@@ -1155,6 +1155,14 @@ MethodTypeForm form() {
 	MethodType asCollectorType(Class<?> clz, int num1, int num2) {
 		throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
 	}
+
+	/**
```

Comment:
The preprocessor tag around this is `[IF Sidecar19-SE-OpenJ9]` - shouldn't there be a new one for Java 10?

Reply:
Fixed. Still use existing JPP tag `Java18.3`, will submit another PR to rename it to `Java10`.